README.md

```diff
@@ -224,9 +224,11 @@ as that file will be overridden on updates.
 
 ### Copyright Checks
 
-By default Baseline enforces Palantir copyright at the beginning of files. To change this, edit the template copyright
-in `.baseline/copyright/*.txt` and the RegexpHeader checkstyle configuration in `.baseline/checkstyle/checkstyle.xml`
+Baseline enforces Palantir copyright at the beginning of files when applying `com.palantir.baseline-format`. To change this, edit the template copyright
+in `.baseline/copyright/*.txt`.
+Note that only the first file (sorted lexicographically) is chosen as the copyright for the entire project.
 
+To automatically update all files with mismatching/missing copyrights, run `./gradlew format`.
```

Comment:
`./gradlew format` feels a bit odd here. I think I'd almost recommend `spotlessApply` here

Reply:
Isn't it the same thing though?

Reply:
Recommending another command that `format` intentionally aliases feels like it might confuse people.